src/run_releasenotes_check.py

```diff
@@ -23,18 +25,41 @@
 def main() -> int:
     args = ReleaseNotesCheckArgs()
     console.configure(level=args.logging_level)
-    manifest_file = InputManifest.from_file(args.manifest)
-    BUILD_VERSION = manifest_file.build.version
-
+    manifests: List[InputManifest] = []
     # storing temporary release notes for testing purposes
     BASE_FILE_PATH = "release_notes_workflow/results"
+
+    for input_manifests in args.manifest:
+        manifests.append(InputManifest.from_file(input_manifests))
+
+    if len(args.manifest) == 2:
```

Comment:
Do you have a check to ensure user input max of 2 manifests? What if someone entered more than 2?

Reply:
nvm, missed the check on more than 2.